govcd/adminvdc_nsxt_test.go

```diff
@@ -91,6 +91,7 @@ func (vcd *TestVCD) Test_CreateNsxtOrgVdc(check *C) {
 		if allocationModel == "Flex" {
 			vdcConfiguration.IsElastic = &trueValue
 			vdcConfiguration.IncludeMemoryOverhead = &trueValue
+			vdcConfiguration.ResourceGuaranteedMemory = addrOf(1.00)
```

Comment:
Would be nice to have a comment here why/in which API version it became needed :)

Reply:
Added